internal/buffer/settings.go

```diff
@@ -117,6 +120,13 @@ func (b *Buffer) DoSetOptionNative(option string, nativeValue interface{}) {
 	if b.OptionCallback != nil {
 		b.OptionCallback(option, nativeValue)
 	}
+
+	_, err := config.RunPluginFnBool(b.Settings, "onBufferOptionChanged",
```

Comment:
Why Bool?

Reply:
Since the boolean result is ignored it doesn't seem to be necessary, except the other small difference that `RunPluginFnBool()` checks for buffer local `settings[p.Name]` being unset, which `RunPluginFn()` doesn't. Therefore with `RunPluginFnBool()` the callback isn't called in plugins which are disabled after loading, while with `RunPluginFn()` it is...which is an inconsistency on his own.